common/src/main/scala/gu/msnotifications/NotificationHubClient.scala

```diff
@@ -15,11 +16,13 @@ object NotificationHubClient {
  * https://msdn.microsoft.com/en-us/library/azure/dn223264.aspx
  */
 final class NotificationHubClient(notificationHubConnection: NotificationHubConnection, wsClient: WSClient)
-                                 (implicit executionContext: ExecutionContext) {
+                                 (implicit executionContext: ExecutionContext) extends NotificationProvider {
 
   import notificationHubConnection._
   import NotificationHubClient.HubResult
 
+  val name = "WNS"
```

Comment:
naming convention for constant vals is to UpperCamelCase so it should be `Name` instead